### PR TITLE
Added in the quadratization of NTR-ABCG-2 example verification

### PR DESCRIPTION
**Pull request by ehuan2**
Added in the quadratization of equation 34 in the reproducing.m file. While testing in matlab, I got ans = 1, which suggests that the LHS and RHS are the same, and so verifies the example. Also added to the Book_about_Quadratization.tex file to show the example's quadratization.

### CODE REVIEW

**Comment by ndattani** [comment]
Thanks for this contribution @ehuan2!

Were you able to compile the LaTeX file? If you were able to do that without getting any references showing up like (???) or [?], then this would be great. You might have to compile several times in a row, as [explained here](https://tex.stackexchange.com/q/53235/14527). Once you get a PDF that looks good, I'd recommend to push the PDF file and the `.log` file into your fork as well (but no other files please). 

If you could also show me a screenshot of what the line that you edited in the PDF looks like, in a comment on this Pull Request, then it would be very helpful for helping me to accept your Pull Request. The PDF viewer on GitHub can be slow, and while on the road I'm not always able to download a large PDF file and open it on the spot, so a screenshot of the one part that changed, would show me that the new version of the book is acceptable, without me having to download the PDF or rely on GitHub's PDF viewer.

Finally, you might have noticed that there's not many branches in this repository, despite us having 15 contributors including yourself. Preferably, you would have made your own fork rather than creating an entirely new branch for this. This was instruction (6) in my email: "Fork the repo and push your changes". If we created a new branch for every addition, the repository would get a bit messy. Now that you've created this branch, you might as well continue using it, but in the future I would recommend to make your own fork first. I also said the same thing to @m6keller who created the NTR-ABCG-1_mkeller branch.

**Comment by ehuan2** [comment]
@ndattani - thanks for the review!

1. I used overleaf online to compile since I can't seem to get it working locally... I downloaded the output log and pdf from there as well.

2. Here's what it looks like:
image

3. Oops, my bad! I will keep the fact of forking the repo in mind for the future.

**Comment by ndattani** [comment]
Somehow your comment was posted exactly at 4:22PM EDT on 23 March 2022, which is **_exactly_** the same time that @johnjeihokim posted their comment [here](https://github.com/HPQC-LABS/Book_About_Quadratization/pull/51#issuecomment-1076783571). 

It looks like both of you were trying to push commits to your respective forks/branches at the same time. Although the emails from you and John _both_ came in my inbox at exactly 4:22pm, John's appeared as the earlier email in my chronologically ordered inbox, so I addressed John's pull request first. That pull request was fully approved and merged into the Master. Now your pull request appears that it can't be merged into Master **_yet_**, because of a conflict (you'd have to pull the latest version from Master, which now include John's latest commits first). 

Also, since you are having trouble compiling the LaTeX code locally, maybe @johnjeihokim can tell you how they did it?

I find it a bit strange that this commit seems to have added to the repo +1152 units and removed 728 units:

image

That is [more than some people did after weeks of contributions](https://github.com/HPQC-LABS/Book_About_Quadratization/graphs/contributors).

Thanks for everything so far!

**Comment by ehuan2** [comment]
Hey @ndattani ! So I went ahead and figured out how to compile locally (was using the wrong tool) and managed to merge and recompile the pdf. I think the reason why I have so many commit differences is because I'm probably using a different system to compile on, so the .log looks quite different (that's where it all comes from!). Let me know if you want me to undo the .log and the .bib file changes!

**Comment by ndattani** [comment]
@ehuan2 Sorry for taking so long to reply to you. It's been a busy week! What was the "wrong tool" you were using, and which tool helped fix it?

I still think the commit differences look a bit large in size, but you have indeed improved it:

image

File: Volume_1/Book_about_Quadratization.tex [hunk 1]
```diff
@@ -1,7 +1,7 @@
 %% LyX 2.2.0 created this file.  For more info, see http://www.lyx.org/.
 %% Do not edit unless you really know what you are doing.
 %\documentclass[english,notitlepage,longbibliography,showpacs,preprintnumbers,amsmath,amssymb,aps,prl,nofootinbib,12pt,superscriptaddress]{revtex4-1}
-\documentclass[english,notitlepage,longbibliography,showpacs,preprintnumbers,amsmath,amssymb,aps,prx,nofootinbib,12pt,superscriptaddress]{revtex4-1}
+\documentclass[english,notitlepage,longbibliography,showpacs,preprintnumbers,amsmath,amssymb,aps,prx,nofootinbib,12pt,superscriptaddress]{revtex4-2}
```
Comment (ndattani) [inline Volume_1/Book_about_Quadratization.tex]: @ehuan2  Why did you have to switch to RevTeX 4.2?

File: Volume_1/Book_about_QuadratizationNotes.bib [hunk 2]
```diff
@@ -1,2 +1,2 @@
-@CONTROL{REVTEX41Control}
-@CONTROL{apsrev41Control,author="00",editor="1",pages="1",title="0",year="0"}
+@CONTROL{REVTEX42Control}
+@CONTROL{apsrev42Control,author="08",editor="1",pages="0",title="0",year="1"}
```
Comment (ndattani) [inline Volume_1/Book_about_QuadratizationNotes.bib]: Why did you have to change author from 00 to 08, and pages from 1 to 0 and year from 0 to 1?

**Comment by ndattani** [comment]
Finally, in the last screenshot you posted in this Pull Request discussion, I'd like you to put "H_2local" on the left of the new equation you added, since this is a 2-local (quadratic) function, not a 6-local one. Also it would be nice if you can number every equation, and also add another line (with a new equation number) for the expanded version of your 2-local Hamiltonian :)

Thanks so much!

**Comment by ndattani** [comment]
Hi @ehuan2, when you get a chance, please try to reply to my messages above!

**Comment by ehuan2** [comment]
yes will do! sorry, it's been a busy week, I'll resolve the merge conflicts + reply above!

File: Volume_1/Book_about_Quadratization.tex (hunk 1 above)
Comment (ehuan2) [reply Volume_1/Book_about_Quadratization.tex]: yeah it's weird, I think it didn't compile before locally with 1, but I can revert it (worked online with overleaf)

File: Volume_1/Book_about_QuadratizationNotes.bib (hunk 2 above)
Comment (ehuan2) [reply Volume_1/Book_about_QuadratizationNotes.bib]: yeah again, it's just weird compilation locally, maybe some dependencies are different? I'll revert it!

**Comment by ehuan2** [comment]
> @ehuan2 Sorry for taking so long to reply to you. It's been a busy week! What was the "wrong tool" you were using, and which tool helped fix it?
> 
> I still think the commit differences look a bit large in size, but you have indeed improved it:
> 
> image

yeah I think this again was the .log file being changed by my local compiler. Not sure why it's so different... I'll revert the log file. I forget what I was using before, it was some combination of pdfLatex and BibTex I think (I can recompile with lots of different ones). I'm using just pdfLatex right now.

**Comment by ehuan2** [comment]
image
@ndattani this is what it looks like now! I also just got 10 changes, let me know if this looks good to you!

**Comment by ndattani** [comment]
Thanks @ehuan2 !  I'm impressed that you got it down to just +10 and -1 changes! One final thing you can do is to add Eq. 36 (the expanded version of H_2local that you just added to the PDF) to `reproducing.m`, so that the equation is also verified in MATLAB.

**Comment by ehuan2** [comment]
> Thanks @ehuan2 ! I'm impressed that you got it down to just +10 and -1 changes! One final thing you can do is to add Eq. 36 (the expanded version of H_2local that you just added to the PDF) to `reproducing.m`, so that the equation is also verified in MATLAB.

Sounds good! Let me know if this looks good to you and I can merge it! (it produces an output of 1 for the answer).

**Comment by ndattani** [comment]
@ehuan2 Excellent! This is ready to get merged into Master, once the (new) conflicts are resolved. The reason for these new conflicts is probably the recent merging of John Kim's branch into Master. Usually I'd wait until your feature is fully merged into Master for this, but I've already added your name to the list of contributors [here](https://github.com/HPQC-LABS/Book_About_Quadratization/blob/master/README.md).

**Comment by ehuan2** [comment]
> @ehuan2 Excellent! This is ready to get merged into Master, once the (new) conflicts are resolved. The reason for these new conflicts is probably the recent merging of John Kim's branch into Master. Usually I'd wait until your feature is fully merged into Master for this, but I've already added your name to the list of contributors [here](https://github.com/HPQC-LABS/Book_About_Quadratization/blob/master/README.md).

great, thank you! I'll merge it now in that case :D

**Pull request merged**